microsoft-365/compliance/alert-policies.md

```diff
@@ -162,7 +162,7 @@ The tables also indicate the Office 365 Enterprise and Office 365 US Government
 
 |Name|Description|Severity|Automated investigation|Enterprise subscription|
 |---|---|---|---|---|
-|**A potentially malicious URL click was detected**|Generates an alert when a user protected by [Safe Links](/microsoft-365/security/office-365-security/safe-links) in your organization clicks a malicious link. This alert is generated when a user clicks on a link and this event triggers a URL verdict change identification by Microsoft Defender for Office 365. This alert automatically triggers [automated investigation and response in Office 365](/microsoft-365/security/office-365-security/office-365-air). For more information on events that trigger this alert, see [Set up Safe Links policies](/microsoft-365/security/office-365-security/set-up-safe-links-policies).|High|Yes|E5/G5 or Defender for Office 365 P2 add-on subscription|
+|**A potentially malicious URL click was detected**|Generates an alert when a user protected by [Safe Links](/microsoft-365/security/office-365-security/safe-links) in your organization clicks a malicious link. This alert is generated when a user clicks on a link and this event triggers a URL verdict change identification by Microsoft Defender for Office 365. It will also check for any clicks in the past 48-hours from the time the malicious URL verdict is identified, and generate alerts for the clicks happened in the 48-hour timeframe for that malicious link. This alert automatically triggers [automated investigation and response in Office 365](/microsoft-365/security/office-365-security/office-365-air). For more information on events that trigger this alert, see [Set up Safe Links policies](/microsoft-365/security/office-365-security/set-up-safe-links-policies).|High|Yes|E5/G5 or Defender for Office 365 P2 add-on subscription|
```

Comment:
```suggestion
|**A potentially malicious URL click was detected**|Generates an alert when a user protected by [Safe Links](/microsoft-365/security/office-365-security/safe-links) in your organization clicks a malicious link. This alert is generated when a user clicks on a link and this event triggers a URL verdict change identification by Microsoft Defender for Office 365. It will also check for any clicks in the past 48 hours from the time the malicious URL verdict is identified, and generate alerts for the clicks that happened in the 48-hour timeframe for that malicious link. This alert automatically triggers [automated investigation and response in Office 365](/microsoft-365/security/office-365-security/office-365-air). For more information on events that trigger this alert, see [Set up Safe Links policies](/microsoft-365/security/office-365-security/set-up-safe-links-policies).|High|Yes|E5/G5 or Defender for Office 365 P2 add-on subscription|
```